<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>871</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>-40</x>
       <y>-20</y>
       <width>1201</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/1/banner.png&quot;/&gt;&lt;img src=&quot;:/banner/ProgramaAluguel/banner.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>1151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QComboBox" name="combo_estado">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AL</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>DF</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ES</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MT</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MG</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PA</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PB</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>PI</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RJ</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RN</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RO</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RR</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SC</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SE</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>TO</string>
      </property>
     </item>
    </widget>
    <widget class="QLineEdit" name="edt_cidade">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edt_bairro">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_quartos">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>10</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_banheiros">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_garagem">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>70</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_preco_min">
     <property name="geometry">
      <rect>
       <x>731</x>
       <y>30</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>999999.989999999990687</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="spin_preco_max">
     <property name="geometry">
      <rect>
       <x>731</x>
       <y>60</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <double>99999.000000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_area_min">
     <property name="geometry">
      <rect>
       <x>591</x>
       <y>30</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>99999</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_area_max">
     <property name="geometry">
      <rect>
       <x>591</x>
       <y>60</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_tipo">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Apartamento</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Casa</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cobertura</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Chácara</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Flat</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Kitnet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Lote</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Estado</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Cidade</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Bairro</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Área Mínima</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>60</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Área Máxima</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>30</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Preço Mínimo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>60</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Preço Máximo</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>10</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Quartos</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>40</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Banheiros</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>830</x>
       <y>70</y>
       <width>47</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Garagem</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_adcionar">
     <property name="geometry">
      <rect>
       <x>1009</x>
       <y>5</y>
       <width>121</width>
       <height>70</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_remover">
     <property name="geometry">
      <rect>
       <x>1069</x>
       <y>75</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_editar">
     <property name="geometry">
      <rect>
       <x>1010</x>
       <y>75</y>
       <width>61</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Editar</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>10</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Tipo de Imóvel</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>1151</width>
      <height>641</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QTableWidget" name="table_imoveis">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1151</width>
       <height>631</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>40</number>
     </attribute>
     <column>
      <property name="text">
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Endereço</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Bairro</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Cidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Estado</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quartos</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Banheiros</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Garagem</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Imóvel</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Área</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Aluguel</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../banner.qrc"/>
 </resources>
 <connections/>
</ui>
